launcher/plans/translator_nest_to_tvb_on_local.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<settings>
+    <co_simulation_action_plan>
+        <title>Test the translator Nest to TVB</title>
+        <description>Test the translator Nest to TVB</description>
+        <parameters>
+            <par_000>
+                <par_name>CO_SIM_XML_ACTIONS_DIR</par_name>
+                <par_value>${CO_SIM_TVB_NEST}/launcher/actions</par_value>
+            </par_000>
+            <par_001>
+                <par_name>CO_SIM_EXECUTION_ENVIRONMENT</par_name>
+                <par_value>Local</par_value>
+            </par_001>
+        </parameters>
+        <variables>
+            <!-- The variables under this section, will be created dynamically and held on a python dictionary
+                 NOTE: the NAME of the variable is the dictionary KEY to have access to its value
+            -->
```

Comment:
I would suggest to add some remarks about '_dynamically created_' here.  Do the variables will be created during run time?